plugins/csl-tools/src/logger.cpp

```diff
@@ -0,0 +1,24 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//                               This file is part of CosmoScout VR                               //
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// SPDX-FileCopyrightText: German Aerospace Center (DLR) <cosmoscout@dlr.de>
+// SPDX-License-Identifier: MIT
+
+#include "logger.hpp"
+
+#include "../../../src/cs-utils/logger.hpp"
+
+namespace csl::tools {
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+spdlog::logger& logger() {
+  static auto logger = cs::utils::createLogger("csl-tools");
+  logger->info("{}", reinterpret_cast<size_t>(logger.get()));
+  return *logger;
```

Comment:
This was nice for testing :wink: 
```suggestion
  static auto logger = cs::utils::createLogger("csl-tools");
  return *logger;
```